### ISSUE TITLE
Crash after loading back into world after freeze

### ISSUE THREAD

**Issue by GrandMageVivi** [anegostudios/VintageStory-Issues#478]
**Game Version:** 1.12.8
**Platform:** Windows
**Modded:** Yes (Carrying Capacity)

### Description
Not really sure what's happening. I left the game unfocused for about 20 minutes (unsure of what the technical term is - I have the game in fullscreen mode and hit escape to bring up my cursor, then clicked on a video in a web browser on my second monitor to pause it whilst I left. When I came back and clicked back into the game, it froze and I had to force quit it. This usually happens when I leave it for longer than around 5 minutes.) Then when loaded back into my world, I was rolled back maybe 5 minutes, which also usually happens when it freezes and I have to force close it.

This time, however, I can't seem to be able to play for longer than a minute or so. After that time, the game crashes and throws up this log. 

I've tried disabling carrying capacity, disabling fullscreen, changing some graphical settings, moving to 1.12.7. and moving to the unstable 1.12.9. but all with the same result. I was really enjoying my world so I'd hate to lose it, just wondering if there's something else I can try,

### How to reproduce
1. Leave game unfocused for long period of time.
2. Come back to game, game freezes.
3. Force quit game then reopen.
3. Load back into world, rolled back.
4. Move around for a bit and crash,

### Logs
```
Running on 64 bit Windows with 0 GB RAM 
Version: v1.12.8 (Stable)3/18/2020 4:11:16 PM: Critical error occurred
System.IO.EndOfStreamException: Unable to read beyond the end of the stream.
   at System.IO.__Error.EndOfFile()
   at System.IO.MemoryStream.InternalReadInt32()
   at Vintagestory.API.Common.EntityAgent.FromBytes(BinaryReader reader, Boolean forClient)
   at Vintagestory.Client.NoObf.ClientSystemEntities.HandleEntitySpawnPacket(_9J05IwqX3DMJjYc4TgQdHtYCjUh serverpacket)
   at _DheU0S41ys8PmWwanVxliVufAfL._8pFQWKHqjk1Z0lk74wVtfciws7e(Single )
   at _EBtxK36Ixmcei9M4cHb3yN0CdBY._SBmZIE5zeKBZLld4sDoBkph45oC(Single )
   at _TCuegFvuCG7xQqhWKljGWgaccAS._xnJNZgMqo5oGjGjHTS2kXgpC9eH(Single )
   at _TCuegFvuCG7xQqhWKljGWgaccAS._uw3xmExX4CRVc5TYKty6OJkT52C(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _BjgcNpIFkj1jqrVxzsqeX8AsdOJ._Sxh5j8nPFBKHENmnO5xs7MH8hze(_dWKuoESsZrjrNSOe8XL3edrjaTJ , String[] )
   at _OeweVW96y8iGyqsCmuTDn7vCxyG._Sxh5j8nPFBKHENmnO5xs7MH8hze(ThreadStart )
-------------------------------

```

**Comment by GrandMageVivi** [comment]
Update: I opened the world in repair mode and flew around where I was crashing and that seemed to fix it for now.

**Comment by Bloodsaw** [comment]
Seems to be a duplicate of savegame corruption #390 cause by known issue #330 . 
I'm glad to hear that you were able to repair your savegame.
@Tyronx since hes able to replicate the pause-freeze, maybe he's a candidate for further investigations.

**Comment by tyronx** [comment]
@Bloodsaw The pause-freeze issue should be fixed already in v1.12.9-rc.1
